addons/hr_timesheet/models/project.py

```diff
@@ -244,6 +244,7 @@ class Task(models.Model):
     effective_hours = fields.Float("Hours Spent", compute='_compute_effective_hours', compute_sudo=True, store=True)
     total_hours_spent = fields.Float("Total Hours", compute='_compute_total_hours_spent', store=True, help="Time spent on this task and its sub-tasks (and their own sub-tasks).")
     progress = fields.Float("Progress", compute='_compute_progress_hours', store=True, group_operator="avg")
+    progress_normalized = fields.Float("Progress (%)", compute='_compute_progress_normalized', store=True, group_operator="avg")
```

Comment:
This is kind of duplicating the 'progress' field information in the db.
Can't we just display `(<field name="percentage" nolabel="1" > %)` in the view?

Reply:
I suppose you meant `name="progress"` on your second line. I did not directly use this field because of the functional specs (i.e. having the progress percentage rounded). As:
- the widget timesheet_uom_no_toggle does only round the percentage if the decimal value is .00
- the field 'progress' is stored in % to be able to be used with this widget
- the standard widget 'percentage' does not accept field stored in %

I thought that the creation of a new computed field was the best option. Although it is not necessary to store it (I have modified that in my code).

Reply:
Yes, `progress` is what I meant! You won't have the decimals like this :
`<span >(<field name="progress" class="oe_inline" nolabel="1" widget="integer"/> %)</span>`

Reply:
Good one, thank you! I updated the branch.